packages/framework/src/Resources/translations/messages.cs.po

```diff
@@ -34,6 +34,9 @@ msgstr "-- Vyberte značku --"
 msgid "-- Choose unit --"
 msgstr "-- Vyberte jednotku --"
 
+msgid "... expand ..."
+msgstr ""
+
```

Comment:
Missing czech translations

Reply:
😱
Thanks for noticing! fixed in b57728196

Reply:
translations no longer necessary